api/src/test/java/org/openmrs/module/registrationcore/RegistrationCoreUtilTest.java

```diff
@@ -2,11 +2,14 @@
 
 import org.joda.time.DateTime;
 import org.junit.Test;
+import org.openmrs.GlobalProperty;
+import org.openmrs.api.context.Context;
+import org.openmrs.web.test.BaseModuleWebContextSensitiveTest;
 
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertThat;
 
-public class RegistrationCoreUtilTest {
+public class RegistrationCoreUtilTest extends BaseModuleWebContextSensitiveTest {
```

Comment:
@dkayiwa The call that requires the Context is inside the implementation of the ResourceCoreUitl method